Tools/build/generate-build-details.py

```diff
@@ -55,7 +55,7 @@ def generate_data(schema_version: str) -> collections.defaultdict[str, Any]:
     data['language']['version'] = sysconfig.get_python_version()
     data['language']['version_info'] = version_info_to_dict(sys.version_info)
 
-    data['implementation'] = vars(sys.implementation)
+    data['implementation'] = vars(sys.implementation).copy()
```

Comment:
this still seems incorrect for cross-builds, and I'm not sure I understand what's the benefit of copying here

Reply:
cross builds: I think off topic for this PR? I think you opened an issue for this already IIRC.